home/models.py

```diff
@@ -22,6 +22,15 @@ class Group(models.Model):
     def __str__(self):
         return self.name
 
+    def number_users(self):
```

Comment:
As the function has no parameters, what do you think of this be a @property?

Reply:
It's fine, I forgot.